### DISCUSSION TITLE
Is it possible to build resource tools without separate compilation?

### DISCUSSION THREAD

**Discussion by kennyray** [laravel/nova-issues#3651] [Q&A]
I followed this tutorial, but it does not seem to work with the latest version of Nova. Does anyone know how to build tools or cards without needing a separate compilation? We do not intend to distribute what we are building and the extra overhead seems unnecessary if we can just compile in our main Laravel app. https://blog.richartkeil.com/custom-laravel-nova-cards/

**Comment by silentpatrik** [comment]
Did you ever find anything related to this?

**Reply by kennyray** [reply]
Sorry to say, I haven’t. We pretty much figure that we will need to build some tools outside Nova.

**Reply by silentpatrik** [reply]
> Sorry to say, I haven’t. We pretty much figure that we will need to build some tools outside Nova.

So, like iframing or including blade views instead? Any link? 

Actually, now when thinking of it; among the laravel web api routes, its should be easy to render novas layout template, maybe woth some pregreplace. Then in the right content area, replace with tool blade views which would give a quick workaround atleast to test stuff locally.

**Comment by kennyray** [comment]
For our project we will probably just build a tool into the system and use the Laravel gate to allow the same users the way Nova does. We haven’t built it yet. 

Kenny
